### PR TITLE
Translate README to German

### PR DESCRIPTION
**Pull request by ParadoxSpiral**
This translation takes into account the edit made to the Running section in #1317.

### CODE REVIEW

**Comment by SirCmpwn** [comment]
Please add details on availability of support in German. Can you volunteer to provide support in the IRC channel for German users? Either way, mention it in the README.

**Comment by SirCmpwn** [comment]
In the Japanese readme I said "私の名前はSirCmpwnです。助けのはIRCとGitHubで日本語で頑張ります", which introduces my IRC handle and says I will do my best to help in Japanese on IRC or GitHub. I also said later, regarding the config file: 英語は難しいのばIRCで助けのを尋ねる。￼日本語助けのはUTC-4。Which means "ask in IRC if you have trouble with the english config file, japanese help is available in the UTC-4 timezone".

File: README.de.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+# sway [![](https://api.travis-ci.org/SirCmpwn/sway.svg)](https://travis-ci.org/SirCmpwn/sway) [![Mit fosspay spenden](https://drewdevault.com/donate/static/donate-with-fosspay.png)](https://drewdevault.com/donate?project=4)
+
+"**S**irCmpwn's **Way**land compositor" ist ein i3-kompatibler 
+[Wayland](http://wayland.freedesktop.org/)-Anzeigeserver. Lies die
```
Comment (refacto) [inline README.de.md]: sway ist kein Anzeigeserver, sondern ein Kompositor.

File: README.de.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+    sudo chmod a+s /usr/local/bin/sway
+
+## Konfiguration
+
+Falls du i3 benutzt, kopiere deine i3 Konfiguration, andernfalls die Beispielkonfiguration, 
+nach `~/.config/sway/config`. Die Beispielkonfiguration sollte `/etc/sway/config` sein.
+Führe `man 5 sway` für Konfigurationsinformationen aus.
+
+Als Inspiration sind meine eigenen Punktdateien [hier](https://git.sr.ht/~sircmpwn/dotfiles)
```
Comment (refacto) [inline README.de.md]: s/Punktdateien/Konfigurationsdateien/

File: README.de.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+nach `~/.config/sway/config`. Die Beispielkonfiguration sollte `/etc/sway/config` sein.
+Führe `man 5 sway` für Konfigurationsinformationen aus.
+
+Als Inspiration sind meine eigenen Punktdateien [hier](https://git.sr.ht/~sircmpwn/dotfiles) 
+verfügbar, schau dir auch unbedingt das [Wiki](https://github.com/SirCmpwn/sway/wiki) an.
+
+## Verwendung
+
+Führe `sway` von einer TTY aus. Manche Anzeige-manager könnten funktionieren, werden aber
+nicht von Sway unterstützt (gdm scheint relativ gut zu funktionieren).
```
Comment (refacto) [inline README.de.md]: Anstatt `startx` auszuführen, führ `sway` aus. Du kannst `sway` auch innerhalb
X ausführen, was nützlich ist um zu testen. Manche Anzeige-manager könnten funktionieren, werden aber
nicht von Sway unterstützt (gdm scheint relativ gut zu funktionieren).

Comment (SirCmpwn) [inline README.de.md]: See the change to the english readme in https://github.com/SirCmpwn/sway/pull/1317

Comment (refacto) [inline README.de.md]: Okay, disregard this specific suggestion.

**Comment by ParadoxSpiral** [comment]
@refacto Thanks for the review! (I also included a lot of changes based on a review by @minus7)

**Comment by SirCmpwn** [comment]
Can you sync with master and update the links at the top of each README?

**Comment by refacto** [comment]
Regarding support: I may be able to provide some support in #sway when I have the time and if there's a person needing support in German.

**Comment by SirCmpwn** [comment]
Are you able to idle in the channel?

**Comment by refacto** [comment]
Yup!

**Comment by SirCmpwn** [comment]
Cool! What's your nick?

**Comment by refacto** [comment]
#azarus

**Comment by SirCmpwn** [comment]
Danke!

**Pull request merged**